File: Service_eHospital_DongNai_A/Implement/BENHNHAN_IMAGESService.cs
namespace Service_eHospital_DongNai_A.Implement
{
    public class BENHNHAN_IMAGESService : BaseService<BENHNHAN_IMAGES, IBENHNHAN_IMAGESRepository>
    , IBENHNHAN_IMAGESService
    {
    private readonly IBENHNHAN_IMAGESRepository _BENHNHAN_IMAGESRepository;
    public BENHNHAN_IMAGESService(IBENHNHAN_IMAGESRepository BENHNHAN_IMAGESRepository) : base(BENHNHAN_IMAGESRepository)
    {
    _BENHNHAN_IMAGESRepository = BENHNHAN_IMAGESRepository;
    }
    }
    }

File: CRM/src/app/shared/eHospital_DongNai_A/BENHNHAN_IMAGES.service.ts
import { Injectable } from '@angular/core';
import { HttpClient, HttpParams } from '@angular/common/http';
import { environment } from 'src/environments/environment';
import { BENHNHAN_IMAGES } from './BENHNHAN_IMAGES.model';
import { BaseService } from './Base.service';
@Injectable({
    providedIn: 'root'
})
export class BENHNHAN_IMAGESService extends BaseService{
    DisplayColumns001: string[] = ['Save', 'STT', 'BenhNhan_Images_Id', 'BenhNhan_Id', 'File_Name_Origin', 'File_Name', 'NhomDichVu_Id', 'TenHinh', 'MoTa', 'NgayGhiNhanKetQua', 'NgayTao', 'NguoiTao_Id', 'NgayCapNhat', 'NguoiCapNhat_Id', 'TiepNhan_Id', 'LoaiHinhAnh_Id', 'Folder', 'NgayFolder'];

    List: BENHNHAN_IMAGES[] | undefined;
    ListFilter: BENHNHAN_IMAGES[] | undefined;
    FormData!: BENHNHAN_IMAGES;
    constructor(public httpClient: HttpClient) {
        super(httpClient);
        this.Controller = "BENHNHAN_IMAGES";
    }
}

